EqualityInformationApi/V1/Gateways/IEqualityInformationGateway.cs

```diff
@@ -11,5 +11,7 @@ namespace EqualityInformationApi.V1.Gateways
     public interface IEqualityInformationGateway
     {
         Task<EqualityInformation> Create(EqualityInformationObject request);
+
+        Task<EqualityInformation> Patch(PatchEqualityInformationObject request);
```

Comment:
I wouldn't use `Patch` internally in code, it's HTTP terminology

Reply:
Update?